src/main/resources/assets/twilightforest/lang/ru_RU.lang

```diff
@@ -1,413 +1,444 @@
-achievement.twilightProgressYeti.desc=Линия одежды с мехом Альмо-йети сохранит вас от мороза
-achievement.twilightProgressGlacier=Чистое небо
-achievement.twilightProgressGlacier.desc=Победите Снежную королеву во дворце Авроры, очистив нагорье от кислотного дождя.
-achievement.twilightProgressTroll=Я желаю больше огня
-achievement.twilightProgressTroll.desc=Найдите лампу огня в пещерах троллей под небесным островом гигантов и она поможет вам.
-achievement.twilightProgressThorns=Мимо шипов
-achievement.twilightProgressThorns.desc=Откройте дверь в замок.
-achievement.twilightProgressCastle=Этот замок классный
-achievement.twilightProgressCastle.desc=Что может быть ещё в этом замке?
-enchantment.tfChillAura=Ледяная аура
-item.minotaurAxe.tooltip=Дополнительный урон во время бега
-item.knightlySword.tooltip=Дополнительный урон по бронированным целям
-item.knightlyPick.tooltip=Дополнительный урон по бронированным целям
-item.knightlyAxe.tooltip=Дополнительный урон по не бронированным целям
-item.giantPick.name=Кирка гиганта
-item.giantSword.name=Меч гиганта
-item.tripleBow.name=Тройной лук
-item.seekerBow.name=Лук искателя
-item.iceBow.name=Ледяной лук
-item.enderBow.name=Эндер лук
-item.iceSword.name=Ледяной меч
-item.glassSword.name=Стеклянный меч
-tile.TrollSteinn.name=Тролльштеин
-tile.WispyCloud.name=Прозрачное облако
-tile.FluffyCloud.name=Взбытое облако
-tile.GiantCobble.name=Гигантский булыжник
-tile.GiantLog.name=Гигантское дерево
-tile.GiantLeaves.name=Гигантские листья
-tile.GiantObsidian.name=Гигантский обсидиан
-tile.UberousSoil.name=Анбеоновая почва
-tile.HugeStalk.name=Огромный стебель
-tile.HugeGloomBlock.name=Огромный грибрил
-tile.TrollVidr.name=Тролловидр
-tile.UnripeTrollBer.name=Спелый тролльбер
-tile.TrollBer.name=Тролльбер
-tile.TFLeaves3.1.name=Бобовые листья
-item.magicBeans.name=Волшебные бобы
-entity.TwilightForest.Stable Ice Core.name=Стабильное замёрзшее ядро
-entity.TwilightForest.Unstable Ice Core.name=Нестабильное замёрзшее ядро
-entity.TwilightForest.Snow Queen.name=Снежная королева
-entity.TwilightForest.Troll.name=Пещерный тролль
-entity.TwilightForest.Giant Miner.name=Гигант шахтёра
-entity.TwilightForest.Armored Giant.name=Бронированный гигант
-entity.TwilightForest.Ice Crystal.name=Ледяной кристалл
-tile.TFFireJet.1.name=Приручённый дымок
-tile.TFFireJet.2.name=Приручённый дымок
-tile.TFFireJet.11.name=Заключённый огонёк
-tile.TFFireJet.12.name=Заключённый шумящий огонёк
-tile.TFFireJet.13.name=Заключённый пылающий огонёк
-item.fieryPick.tooltip=Авто переплавка
-tem.giantPick.tooltip=Ломает гигантские блоки
-item.fierySword.tooltip=Поджигает цель
-item.fieryHelm.tooltip=Поджигает атакующего
-item.fieryPlate.tooltip=Поджигает атакующего
-item.fieryLegs.tooltip=Поджигает атакующего
-item.fieryBoots.tooltip=Поджигает атакующего
-item.yetiHelm.tooltip=Охлаждает атакующего
-item.yetiPlate.tooltip=Охлаждает атакующего
-item.yetiLegs.tooltip=Охлаждает атакующего
-item.yetiBoots.tooltip=Охлаждает атакующего
+

```

Comment:
Because of this empty line, its hard to see what exactly was changed. Github just shows as if the whole file was rewritten instead of showing changes line by line.

Reply:
> Because of this new extra empty line(?), its hard to see what exactly was changed. Github just shows as if the whole file was rewritten instead of showing changes line by line.

Well, the whole file was actually rewritten. That's because I didn't use your repository when I started working on the mod. Try to compare my translation to en_EN file and you'll see that the translation is much more correct than it used to be (it should be easy, because I sorted the lines in file in the exact same order as they were in the original)